src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -192,40 +216,44 @@ class HoldRequest extends React.Component {
     return '';
   }
 
-  /**
-   * renderEDD()
-   * Renders the radio input fields of EDD.
-   *
-   * @return {HTML Element}
-   */
-  renderEDD() {
-    return (
-      <label
-        className="electronic-delivery"
-        id="radiobutton-group1_electronic-delivery"
-        htmlFor="available-electronic-delivery"
-      >
-        <input
-          aria-labelledby="radiobutton-group1 radiobutton-group1_electronic-delivery"
-          type="radio"
-          name="delivery-location"
-          id="available-electronic-delivery"
-          value="edd"
-          checked={this.state.checkedLocNum === -1}
-          onChange={e => this.onRadioSelect(e, -1)}
-        />
-        Have up to 50 pages scanned and sent to you via electronic mail.
-      </label>
-    );
+  // redirects if patron is ineligible to place holds
+
+  conditionallyRedirect() {
+    return this.checkEligibility(this.state.patron.id).then((eligibility) => {
```

Comment:
Here you call `this.checkEligibility` and pass in `this.state.patron.id` but the `checkEligibility` function does not take in a parameter.

Reply:
I see, now, that you don't use the front-end's state at all for the patron's id but instead use what's stored in jwt in the backend. You should make a note of that.

Reply:
Fixed